### ISSUE TITLE
SEO: Add a sitemap and point Google Search Console to it

### ISSUE THREAD

**Issue by hXtreme** [cis547/cis547.github.io#44]
## Adding sitemap

A sitemap can be auto generated using Jekyll plugins to make it easier to manage and update 

See: https://github.com/jekyll/jekyll-sitemap

## Adding sitemap to Google search console

Google can better index and serve the website in search results if it has a sitemap as it allows Google to resolve canonical paths of various pages on the website.

Once a sitemap is generated, talk to @hXtreme to set up the Google search console

**Comment by hXtreme** [comment]
@brmassey05 I'll let you prioritize/de-prioritize this, lmk know if you need any help or more details.

**Comment by k3sero** [comment]
I am having a problem with sitemap.xml and the Google Console Search. I am trying to upload the path /sitemap.xml and Google says that is "unknow" and doesnt charge any url of my website.

The url is correct and contains url valids. Any help?

**Comment by hXtreme** [comment]
@k3sero hi, is this about the cis547 course webpage?

**Comment by k3sero** [comment]
No, its about my sitemap and google console Search. GCS report me that its unavaible connect with my sitemap. 

My blog is k3sero.github.io and my sitemap is k3sero.github.io/sitemap.xml generated with the seo jekyll plugin (its correct)
